mlpp_lib/datasets.py

```diff
@@ -45,8 +44,8 @@ class DataModule:
         and targets are lists of names.
     filter: `DataFilter`, optional
         The object that handles the data filtering.
-    standardizer: `Standardizer`, optional
-        The object to standardize data, already fitted on the training data.
+    data_transformer: `DataTransformer`, optional
```

Comment:
I like the DataTransformer idea, but in here I would still call the input variable "normalizer" since it has a clear scope: normalize the data for the training.

Reply:
I disagree, the whole point of calling it data_transformer is that we don't necessarily _normalize_ the data anymore. Right now there are only _Normalizer_ objects in the data_transformer but in the future, there could be another class _Transformer_ which handles transformations of the data rather than normalization.
By the way, looking at that, it could be relevant the re-change the DataTransformation back to a DataNormalization, what do you think ? That way, we can clearly separate between normalizations and transformations (DataTransformer would be agnostic to that difference since it's outside the class).